<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="HPaf008ax" ignored="2" name="Test1" tests="1" failures="1" timestamp="2020-10-22T18:12:16 IST" time="10.92" errors="0">
  <testcase name="verifyTitle" time="0.034" classname="com.guru99.testCases.tc_01_LoginTest">
    <failure type="java.lang.AssertionError" message="Test Failed expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: Test Failed expected [true] but found [false]
at com.guru99.testCases.tc_01_LoginTest.verifyTitle(tc_01_LoginTest.java:56)
at java.util.ArrayList.forEach(Unknown Source)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
... Removed 21 stack frames]]>
    </failure>
  </testcase> <!-- verifyTitle -->
  <testcase name="invalidLoginTest" time="0.0" classname="com.guru99.testCases.tc_01_LoginTest">
    <ignored/>
  </testcase> <!-- invalidLoginTest -->
  <testcase name="validLoginTest" time="0.0" classname="com.guru99.testCases.tc_01_LoginTest">
    <ignored/>
  </testcase> <!-- validLoginTest -->
</testsuite> <!-- Test1 -->
